ext/QuantumToolboxChainRulesCore.jl

```diff
@@ -0,0 +1,14 @@
+module QuantumToolboxChainRulesCore
```

Comment:
Maybe still add `Ext` at the end ?

```suggestion
module QuantumToolboxChainRulesCoreExt
```

although it's a bit long, but I think it's okay cause user don't need to type these extension names.

Reply:
Yes correct